crates/control-interface/src/types.rs

```diff
@@ -418,3 +418,9 @@ pub struct LinkDefinition {
     pub contract_id: String,
     pub values: LinkSettings,
 }
+
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+pub struct HostLabel {
+    pub label: String,
```

Comment:
I'm going back and forth on this being `label` or `key`, did you consider `key` so it's just submitting kv pairs?

Reply:
Hmm I didn't think about this, but I like what you're saying. A "label" is really a key + value

I'll rename 👍